docs/source/notebooks/SMC2_gaussians.ipynb

```diff
@@ -4,7 +4,7 @@
    "cell_type": "markdown",
```

Comment:
<p>The rank plot shows that <s>one</s> each one of the two SMC runs is exploring a different mode, so even SMC is failing to explore both modes <strong>in the same chain and in proportions equal to the true weight of each mode.</strong></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc3/pull/3981/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>